catalog/video/video-mpegts-stream/pom.xml

```diff
@@ -219,22 +219,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.70</minimum>
+                                            <minimum>0.67</minimum>
```

Comment:
Previous comment was for the `StreamMonitorHelper` class but one of those options should also work for `UdpStreamMonitor`.